content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Network Resilience and Testing
+description: Learn about the dynamic failure recovery features of Istio that you can configure to build tolerance for failing nodes, and to prevent cascading failures to other nodes.
+weight: 3
+keywords: [traffic-management, timeout, retry, circuit-breaker, health-check, fault-injection, fault-tolerance-library]
+---
+
+Istio provides opt-in failure recovery features that you can configure
+dynamically at runtime through the [Istio traffic management rules](/docs/concepts/traffic-management/routing/virtual-services/#routing-rules).
```

Comment:
as I noted above, routing rules have (the link you have) have nothing to do with failure recovery. Remove the link and simply state this as traffic management rules.